package.json

```diff
@@ -95,6 +95,7 @@
   },
   "dependencies": {
     "babel-runtime": "6.9.2",
-    "vue": "1.0.24"
+    "vue": "1.0.24",
+    "vue-multiselect": "^1.0.1"
```

Comment:
I don't have a cdnjs or jsdelivr link for vue-multiselect. I used rawgit, I hope it is enough.